Automattic/wp-calypso
# Promote Post Cancel route fix and invalidate caches

#### Proposed Changes
- Fixes "cancel" button routing in promote post widget when there's no previousRoute defined
- Pre-emptively invalidate cache for campaigns when modal is closed
- Fixed some naming in the subtitle of advertising page

![image](https://user-images.githubusercontent.com/43957544/197566910-63392bd4-d12b-4edb-bb3d-ef59772b6c12.png)


#### Testing Instructions

- Go to the promote post page and promote an ad. You can go from any endpoint where we show the "promote" button
- Do a browser hard-refresh (f5)
- Click on "cancel" button. Result: The screen should open the same location where the endpoint was opened


#### Pre-merge Checklist



Related to #
